### ISSUE TITLE
In the unit entries on units.wesnoth.org, put link to the unit cfg (GNA #20283)

### ISSUE THREAD

**Issue by wesnoth-bugs** [wesnoth/wesnoth#1321]
**[Original submission](https://gna.org/bugs/index.php?20283)** by anonymous on 2012-11-03

In the unit entries on units.wesnoth.org, put a link to the unit cfg file. Eg http://units.wesnoth.org/1.10/mainline/en_US/Blood%20Bat.html would have a link to http://svn.gna.org/viewcvs/*checkout*/wesnoth/branches/1.10/data/core/units/bats/Bat_Blood.cfg . Trivial for mainline core & campaign units, more difficult for add-on units.

(Reproduced on yes)
Release: yes
Priority: 5 - Normal
Severity: 1 - Wish

**Comment by AI0867** [comment]
The presence of the URL could be (for add-ons) contingent on a VCS webinterface base URL being provided in the PBL.

**Comment by cooljeanius** [comment]
> The presence of the URL could be (for add-ons) contingent on a VCS webinterface base URL being provided in the PBL.

Yeah, it would be nice if [PblWML](https://wiki.wesnoth.org/Pblwml) provided a valid key for such a purpose...
